src/ui/units/wizard/selectors/index.ts

```diff
@@ -19,3 +21,9 @@ export const addFieldContainerFieldsSelector = createSelector(
     selectHierarchies,
     ({measures, dimensions}, hierarchies) => [...dimensions, ...hierarchies, ...measures],
 );
+
+export const selectWizardWorkbookId = createSelector(
+    selectWidget,
+    selectDataset,
+    (widget, dataset) => (widget.workbookId as string | null) || dataset?.workbook_id || null,
```

Comment:
It may be wrong to take the workbook_id from the dataset, since the user may try to create a chart in his workbook with someone else's dataset

Reply:
fixed – https://github.com/datalens-tech/datalens-ui/pull/503/commits/4fe80fb0a34680b9643bc0828a0f2ed4e2edf9f4